pandas/io/clipboard.py

```diff
@@ -86,6 +89,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
             obj.to_csv(buf, sep=sep, **kwargs)
             clipboard_set(buf.getvalue())
             return
+        except UnicodeEncodeError:
+            # try again with encoding from locale
+            from locale import getdefaultlocale
+            obj.to_csv(buf, sep=sep, encoding=getdefaultlocale()[1], **kwargs)
```

Comment:
Thanks. I didn't want to submit a new PR while yours is open. But probably more important is whether this works for your edge cases and on your system.